source/Calamari.Scripting/Calamari.Scripting.csproj

```diff
@@ -35,11 +35,11 @@
             <PackageFlatten>false</PackageFlatten>
             <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
         </Content>
-        <Content Include="ScriptCS/*.*">
-            <LinkBase>ScriptCS</LinkBase>
+        <Content Include="scriptcs/*.*">
+            <LinkBase>scriptcs</LinkBase>
             <Visible>false</Visible>
             <Pack>true</Pack>
-            <PackagePath>contentFiles/any/any/ScriptCS/</PackagePath>
+            <PackagePath>contentFiles/any/any/scriptcs/</PackagePath>
```

Comment:
The case change is to match the case of the directory on the file system so that the files are correctly included on case-sensitive file systems.